ortools/linear_solver/xpress_interface.cc

```diff
@@ -1912,16 +1912,10 @@ MPSolver::ResultStatus XpressInterface::Solve(MPSolverParameters const& param) {
 
   int xpress_stat = 0;
   if (mMip) {
-    if (this->maximize_)
-      status = XPRSmaxim(mLp, "g");
-    else
-      status = XPRSminim(mLp, "g");
+    status = XPRSmipoptimize(mLp, "g");
     XPRSgetintattrib(mLp, XPRS_MIPSTATUS, &xpress_stat);
   } else {
-    if (this->maximize_)
-      status = XPRSmaxim(mLp, "");
-    else
-      status = XPRSminim(mLp, "");
+    status = XPRSlpoptimize(mLp, "g");
```

Comment:
you should definitely keep the flags empty here, to keep the same behaviour as before (see doc)
```suggestion
    status = XPRSlpoptimize(mLp, "");
```

Reply:
As far as I understand, 
`MPObjective::SetMinimization` and `MPObjective::SetMaximisation`
call 
`MPObjective::SetOptimizationDirection`
that calls
`interface_->SetOptimizationDirection`
that is overridden by
```c++
void XpressInterface::SetOptimizationDirection(bool maximize) {
  InvalidateSolutionSynchronization();
  XPRSchgobjsense(mLp, maximize ? XPRS_OBJ_MAXIMIZE : XPRS_OBJ_MINIMIZE);
}
```
So for me there is no need to call `XPRSchgobjsense` again.
Can you confirm @flomnes @pet-mit?

Reply:
you're right, sorry, my ctrl+f did not work :p